src/App.tsx

```diff
@@ -1,34 +1,54 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { getTodos } from './api';
+import { actions as setTodosActions } from './features/todos';
 
 export const App: React.FC = () => {
+  const [isLoader, setIsLoader] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
you don't need to specify it, typescript will now that it's a boolean